flight-tracker-client/src/components/NavBar.tsx

```diff
@@ -109,11 +126,13 @@ function NavBar() {
                 </div>
               )}
             </Menu>
-            <p className="toolbar-user">Username</p>
-            <img
-              src="https://via.placeholder.com/50"
-              className="toolbar-avatar"
-            />
+            <p className="toolbar-user">{user !== "" && user}</p>
```

Comment:
user

Reply:
{user !== "" && user} doesn't really make sense as a statement.  Just typing {user} is equivalent.

Reply:
do you want a fallback value? if so {user ?? "Some string you want to display when logged out"}